lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -338,6 +347,27 @@ def write(chunk)
       # Group the entries since we have to make one call per tag.
       grouped_entries = {}
       chunk.msgpack_each do |tag, *arr|
+        tag = tag.to_s
+        if @require_valid_tags
+          # When require_valid_tags is true, invalid cases will cause the log to
```

Comment:
This comment seems outdated and should have been removed in commit d83ef7e already.